### DISCUSSION TITLE
ComfyUI Desktop

### DISCUSSION THREAD

**Discussion by jaivishal0902** [Comfy-Org/ComfyUI#6182] [General]
Hello - I saw some videos of ComfyUI desktop on YouTube and got quite excited to see the ease of use and not running into installation issues. I did sign up for it and on the waitlist. Any idea how long does it take when i would receive the download link? Is there any way, i could jump the queue (stupid question maybe).

**Comment by LukeG89** [comment]
You can download it from the official Desktop repository: https://github.com/Comfy-Org/desktop?tab=readme-ov-file#download

But remember that it's still in beta, so you can experience bugs.

**Reply by jaivishal0902** [reply]
Thanks a ton. Will be better than my manual install i guess which was horrible for me.
